src/irmin-pack/inode_intf.ml

```diff
@@ -152,8 +163,11 @@ module type Sigs = sig
 
   module Make_internal
       (Conf : Conf.S)
-      (H : Irmin.Hash.S)
-      (Key : Irmin.Key.S with type hash = H.t and type t = H.t)
+      (H : Irmin.Hash.S) (Key : sig
+        include Irmin.Key.S with type hash = H.t
+
+        val unfindable_of_hash : hash -> t
```

Comment:
What does unfindable_of_hash do?

Reply:
It is a trick that makes easy the implementation of `portable` inodes